### PR TITLE
vmui: logs explorer

### PR DESCRIPTION
**Pull request by Loori-R**
Added the Logs Explorer page.
This page offers three ways of viewing data: `grouped` by common fields, in a `table view,` and `JSON view.`

Please note, the UI limits the display to a maximum of `1000` logs. This is to maintain performance and prevent overwhelming the interface.

A new build command, `build:logs`, has also been added. This script initiates a build process that activates display of the log page only.

### CODE REVIEW

File: app/vlselect/vmui/dashboards/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Predefined dashboards
```
Comment (tenmozes) [inline app/vlselect/vmui/dashboards/README.md:1]: we don't need that for logs

File: app/vlselect/vmui/dashboards/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+window.__VMUI_PREDEFINED_DASHBOARDS__ = [
```
Comment (tenmozes) [inline app/vlselect/vmui/dashboards/index.js:1]: as abo e

File: app/vlselect/vmui/dashboards/perJobUsage.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (tenmozes) [inline app/vlselect/vmui/dashboards/perJobUsage.json:1]: as above

**Review by tenmozes — CHANGES_REQUESTED** [review]

File: app/vlselect/vmui/dashboards/perJobUsage.json (hunk 3 above)
Comment (dmitryk-dk) [reply app/vlselect/vmui/dashboards/perJobUsage.json]: removed this folder

File: app/vlselect/vmui/dashboards/index.js (hunk 2 above)
Comment (dmitryk-dk) [reply app/vlselect/vmui/dashboards/index.js]: removed this folder

File: app/vlselect/vmui/dashboards/README.md (hunk 1 above)
Comment (dmitryk-dk) [reply app/vlselect/vmui/dashboards/README.md]: removed this folder

**Review by tenmozes — APPROVED** [review]

**Review by dmitryk-dk — APPROVED** [review]

**Pull request merged**